js/server/modules/org/arangodb/foxx/request_context.js

```diff
@@ -466,22 +495,79 @@ extend(RequestContext.prototype, {
 ///   type: FoxxBodyModel
 /// });
 /// ```
+///
+/// ```js
+/// app.post("/foxx", function (req, res) {
+///   var joiBody = req.parameters.joiBody;
+///   // Do something with the number
+/// }).bodyParam(
+///   "joiBody",
+///   joi.number().integer().min(5)
+///   .description("A number greater than five")
+///   .meta({allowInvalid: false}) // default
```

Comment:
It is. But thanks to joi's `meta` method you can just append the options to the schema directly.